src/Table/Table.js

```diff
@@ -100,50 +96,12 @@ export default class Table extends React.Component {
       ).height;
     }
 
-    if (props.useFlex) {
```

Comment:
This looks extremely intimidating. But all of this logic was to determine the table's height, etc. Since the table just flows as long as it needs to, we can safely remove :+1: